metaseq/modules/feedforward_network.py

```diff
@@ -34,16 +34,14 @@ def FeedForwardNetwork(x, fc1, activation_fn, fc2, dropout_module):
         x = x + bias_fc2
     elif model_parallel:
         # here, we do the bias computation inside fc1 and fc2 AND gather_output
-        x, _ = fc1(x)
-        x = activation_fn(x)
+        x = activation_fn(x, fc1(x)[0], True)
```

Comment:
Nit: add `model_parallel=True` here - took me a good minute to figure out what this boolean meant